R/fetch-token.R

```diff
@@ -11,7 +11,7 @@ fetch_token <- function(scopes, ...) {
     # TODO(craigcitro): Expose error handling and/or silencing here.
     try({
       token <- f(scopes, ...)
-    })
+    }, silent = TRUE)
     if (!is.null(token)) {
```

Comment:
This is definitely a net win right now.

The TODO should be:

    TODO: Add flags to make all credential fetch attempts and failures verbose and/or fatal, 
    to help aid debugging when the expected type of credentials isn't being returned.

Happy to send a PR to update, though I'll wait until after this so you don't have a merge conflict. 😉